WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -0,0 +1,294 @@
+    
+    NSMutableAttributedString *commentsText = [[NSMutableAttributedString alloc] initWithString:[NSLocalizedString(@"Comments", @"Stats Comments label") uppercaseStringWithLocale:[NSLocale currentLocale]]];
+    InlineTextAttachment *commentsTextAttachment = [InlineTextAttachment new];
+    commentsTextAttachment.fontDescender = self.todayCommentsLabel.font.descender;
+    commentsTextAttachment.image = commentsImage;
+    [commentsText insertAttributedString:[NSAttributedString attributedStringWithAttachment:commentsTextAttachment] atIndex:0];
+    [commentsText insertAttributedString:[[NSAttributedString alloc] initWithString:@" "] atIndex:1];
+    self.todayCommentsLabel.attributedText = commentsText;
+    self.todayCommentsLabel.textColor = [WPStyleGuide greyDarken20];
+    
+    // Default values for no data
+    self.mostPopularDay.text = @"--";
+    self.mostPopularDay.textColor = [WPStyleGuide greyDarken30];
+    self.mostPopularDayPercentWeeklyViews.text = [NSString stringWithFormat:NSLocalizedString(@"%@ of Weekly Views", @"Insights Percent of weekly views label with value"), @"--"];
+    self.mostPopularDayPercentWeeklyViews.textColor = [WPStyleGuide greyDarken10];
+    self.mostPopularHour.text = @"--";
+    self.mostPopularHour.textColor = [WPStyleGuide greyDarken30];
+    self.mostPopularHourPercentDailyViews.text = [NSString stringWithFormat:NSLocalizedString(@"%@ of Daily Views", @"Insights Percent of daily views label with value"), @"--"];
+    self.mostPopularHourPercentDailyViews.textColor = [WPStyleGuide greyDarken10];
+    self.allTimePostsValueLabel.text = @"--";
+    self.allTimePostsValueLabel.textColor = [WPStyleGuide greyDarken30];
+    self.allTimeViewsValueLabel.text = @"--";
+    self.allTimeViewsValueLabel.textColor = [WPStyleGuide greyDarken30];
+    self.allTimeVisitorsValueLabel.text = @"--";
+    self.allTimeVisitorsValueLabel.textColor = [WPStyleGuide greyDarken30];
+    self.allTimeBestViewsValueLabel.text = @"--";
+    self.allTimeBestViewsValueLabel.textColor = [WPStyleGuide greyDarken30];
+    self.allTimeBestViewsOnValueLabel.text = NSLocalizedString(@"Unknown", @"Unknown data in value label");
+    self.allTimeBestViewsOnValueLabel.textColor = [WPStyleGuide greyDarken10];
+    self.todayViewsValueLabel.text = @"--";
+    self.todayViewsValueLabel.textColor = [WPStyleGuide grey];
+    self.todayVisitorsValueLabel.text = @"--";
+    self.todayVisitorsValueLabel.textColor = [WPStyleGuide grey];
+    self.todayLikesValueLabel.text = @"--";
+    self.todayLikesValueLabel.textColor = [WPStyleGuide grey];
+    self.todayCommentsValueLabel.text = @"--";
+    self.todayCommentsValueLabel.textColor = [WPStyleGuide grey];
+
+    [self retrieveStats];
+}
+
+
+
+#pragma mark - UITableViewDelegate methods
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+    
+    if (indexPath.section == 2) {
```

Comment:
This will be fixed in a PR soon - I'm revamping the controls and will be making individual pieces tappable.